automation_tools/__init__.py

```diff
@@ -353,6 +353,28 @@ def setup_default_capsule(interface=None):
     )
 
 
+def setup_email_notification(smtp=None):
```

Comment:
@elyezer the smtp param would only be an url (there's a separate field in the email.yaml file for authentication), but I'll document it.